Helper/Data.php

```diff
@@ -773,7 +773,6 @@ public function getOneClickPaymentMethods($customerId, $storeId, $grandTotal, $r
 
         $baCollection = $this->_billingAgreementCollectionFactory->create();
         $baCollection->addFieldToFilter('customer_id', $customerId);
-        $baCollection->addFieldToFilter('store_id', $storeId);
```

Comment:
can we omit storing store_id at first place?

Reply:
Yes we can but why?